src/aat/resources/cucumber/relisting.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Re-listing online hearing
+
+  Scenario: Drafting a reason for re-listing
+    Given a standard online hearing is created
+    And the relist reason is set to 'Not ready yet'
+    When drafting the relist
+    Then the online hearing state is 'continuous_online_hearing_started'
+    And the relist state should be 'drafted'
+
+  @wip
+  Scenario: Submitting re-listing reason
+    Given a standard online hearing is created
+    And the relist reason is set to 'Has to be re-listed.'
+    When issuing the relist
+    And the notification scheduler runs
+    And the online hearing state is refreshed
+    Then the online hearing state is 'continuous_online_hearing_relisted'
+    And the relist state should be 'issued'
```

Comment:
1. Also need to test empty reason is acceptable
2. Need to test updating

Reply:
Most of these cases are covered in unit tests.